### ISSUE TITLE
aws-ts-eks example creates duplicate default StorageClasses

### ISSUE THREAD

**Issue by devotee101** [pulumi/examples#673]
Running the aws-ts-eks demo in its current form results in two default gp2 StorageClasses, so running `kubectl get storageclasses` results in the following output:
```
NAME                             PROVISIONER             AGE
cluster-gp2-ab91spdl (default)   kubernetes.io/aws-ebs   34m
gp2 (default)                    kubernetes.io/aws-ebs   36m
```
Once the cluster is created, any deployments requiring a volume fail.

**Comment by devotee101** [comment]
Including the gp2 storage class seems to cause the issue, since EKS creates one already.

https://github.com/pulumi/examples/blob/1d8be5553d2d57dcab9d18fac44da5dbb86e96e2/aws-ts-eks/index.ts#L17

**Comment by stack72** [comment]
closed via #674
